cura/CuraApplication.py

```diff
@@ -942,6 +949,9 @@ def registerObjects(self, engine):
         qmlRegisterSingletonType(ContainerManager, "Cura", 1, 0, "ContainerManager", ContainerManager.getInstance)
         qmlRegisterType(SidebarCustomMenuItemsModel, "Cura", 1, 0, "SidebarCustomMenuItemsModel")
 
+        from cura.API import CuraAPI
+        qmlRegisterSingletonType(CuraAPI, "Cura", 1, 1, "API", self.getCuraAPI)
```

Comment:
We could, but then we'd need to copy the functionality since we'd also need the account to be accessible in both cases. (eg; we want it to be accessible from GUI and for plugins). 

Is there an advantage to doing this that i'm missing?